tsconfig.json

```diff
@@ -30,9 +30,7 @@
     "preserveConstEnums": true,
     "strict": true,
     /* Projects */
-    "incremental": true,
+    "incremental": true
   },
-  "include": [
-    "src/**/*.ts"
-  ]
-}
\ No newline at end of file
+  "include": ["**/src/**/*.ts"]
```

Comment:
Currently with `"src/**/*.ts"` no files are included and TS is complaining. Not sure what was the exact idea here. Perhaps `"include": []` was intended?